lib/converter/rule.go

```diff
@@ -177,7 +177,7 @@ func ruleBackendToAPI(br model.Rule) api.Rule {
 // ruleActionAPIToBackend converts the rule action field value from the API
 // value to the equivalent backend value.
 func ruleActionAPIToBackend(action string) string {
-	if action == "pass" {
+	if action == "Pass" {
```

Comment:
If we're preserving the v1 semantics, we should push things to lower-case in these functions.  If we're not, then you'll need to analyse Felix to make sure it doesn't rely on case.

Reply:
still not expecting any changes in this file - since it's only used by the v1 client (I hope!)